osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -127,13 +129,13 @@ void genDetailsFromAddr(const struct ifaddrs* addr, QueryData& results) {
       }
 
       if (ioctl(fd, SIOCGIFFLAGS, &ifr) >= 0) {
-        r["flags"] = INTEGER(static_cast<size_t>(ifr.ifr_flags));
+        flags |= static_cast<size_t>(ifr.ifr_flags);
```

Comment:
@theopolis It looks to me this code was forgotten here since `r["flags"]` would be overridden anyway, but can you take a look?

Reply:
Is this for the case that the sysfs doesn't exist? Also what is the difference between the ioctl and reading from sysfs?

Reply:
I agree, this seems to be forgotten from the refactor to use sysfs. The | looks correct, to handle sysfs’s limitations mentioned in the PR top-post (promisc+volatile). I am assuming this handles it since it was handled pre-sysfs.